soc/arm64/nxp_imx/mcimx8mm_a53/soc_a53.c

```diff
@@ -16,7 +16,7 @@
  *
  * @return 0
  */
-static int mimx8mm_a53_init(struct device *arg)
+static int mimx8mm_a53_init(const struct device *arg)
```

Comment:
This change should be part of SoC commit.

Reply:
ok, I could squash the cleanup commit.

Reply:
please do so!